compiler/luci/import/src/Nodes/CircleConst.cpp

```diff
@@ -30,10 +30,10 @@
 namespace
 {
 
-std::ostream &operator<<(std::ostream &os, const std::vector<int32_t> &vect)
+std::ostream &operator<<(std::ostream &os, const luci::VectorWrapper<int32_t> &vect)
 {
   uint32_t seq = 0;
-  for (auto &v : vect)
+  for (const auto &v : vect)
```

Comment:
what happens when we don't add `const` ?
I'd like to know if this is necessary for using direct access.

Reply:
Unfortunately, compilation error occurs:

```cpp
/ONE/compiler/luci/import/src/Nodes/CircleConst.cpp: In function ‘std::ostream& {anonymous}::operator<<(std::ostream&, const luci::VectorWrapper<int>&)’:
/ONE/compiler/luci/import/src/Nodes/CircleConst.cpp:36:18: error: cannot bind non-const lvalue reference of type ‘int&’ to an rvalue of type ‘int’
   for (auto &v : vect)
```